lib/hammer_cli_foreman_puppet/host.rb

```diff
@@ -41,7 +40,7 @@ class InfoCommand < HammerCLIForeman::InfoCommand
   HammerCLIForeman::Host::CreateCommand.extend_with(
     HammerCLIForemanPuppet::CommandExtensions::PuppetEnvironment.new
   )
-  HammerCLIForeman::Host::CommonUpdateOptions.extend_with(
-    HammerCLIForemanPuppet::CommandExtensions::PuppetEnvironment.new
-  )
-end
\ No newline at end of file
+  #HammerCLIForeman::Host::CommonUpdateOptions.extend_with(
+  #  HammerCLIForemanPuppet::CommandExtensions::PuppetEnvironment.new
+  #)
```

Comment:
is this reduantant?

Reply:
As far as I can see, this is not needed as there is no such extension in hammer-cli-foreman. Feel free to double-check.